dotenv-linter/src/checks/substitution_key.rs

```diff
@@ -139,4 +143,17 @@ mod tests {
             ],
         );
     }
+
+    #[test]
+    fn check_variable_alternate() {
+        check_test(
+            &mut SubstitutionKeyChecker::default(),
+            [
+                ("FOO=${NODE_ENV:development}", None),
+                ("BAR=${CI:+only}", None),
+                ("BUS=${var:=DEFAULT}", None),
+                ("FOOBAR=${var:?IS_UNSET}", None),
```

Comment:
```suggestion
                ("FOOBAR=${var:?IS_UNSET}", None),
                ("BUZZ=${BAR:-DEFAULT}", None),
```
Let's add another example.